Tests/L2Tests/L2Tests/tests/SystemService_L2test.cpp

```diff
@@ -0,0 +1,444 @@
+#include <gtest/gtest.h>
```

Comment:
Rename SystemService_L2test.cpp to SystemService_L2Test.cpp to match the Pascal case class name.

Reply:
Done